themes/ocw-prototype/assets/sass/main.scss

```diff
@@ -0,0 +1,26 @@
+$course-image-url: "{{ with .Site.GetPage "/" }}{{ .Params.course_image }}{{ end }}" !default;
+
+.header {
+    background-color: #D8D8D8;
+}
+
+.main {
+    background-color: #F0F0F0;
+}
+
+.header div {
+    padding: 0 20px 0 20px;
+}
+
+.btn {
+    background-color: #6F6F6F;
+    color: #FFFFFF;
+}
+
+.jumbotron {
+    background: url($course-image-url) no-repeat top center;
+    -webkit-background-size: cover;
```

Comment:
No need to put this in this PR, but we use autoprefixer to translate `background-size` to add the other prefixes in other projects. It looks like [PostCSS](https://gohugo.io/hugo-pipes/postcss/) has support to integrate autoprefixer with its pipeline. PostCSS requires node.js but maybe we can add that in a `Dockerfile`.

Reply:
True, we should do this.  Netlify CMS will require an npm config anyway.